### PR TITLE
Update Grafana dashboard to SwanLake alpha

### PR DESCRIPTION
**Pull request by tharinduwijewardane**
https://github.com/ballerina-platform/module-ballerinax-prometheus/issues/4

Ballerina Up / Down
Screenshot 2021-02-01 at 10 50 19

Service metrics
Screenshot 2021-02-01 at 10 50 58

Client metrics
Screenshot 2021-02-01 at 10 52 15

Screenshot 2021-02-01 at 10 57 18

Sql client metrics
Screenshot 2021-02-01 at 10 52 35

### CODE REVIEW

File: grafana-dashboard.json [hunk 1]
```diff
@@ -1377,8 +1394,11 @@
       "linewidth": 1,
       "links": [],
       "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
```
Comment (nadundesilva) [inline grafana-dashboard.json:1399]: We should revert this change.

File: grafana-dashboard.json [hunk 2]
```diff
@@ -2819,6 +2871,5 @@
   "timezone": "",
   "title": "Ballerina Metrics",
   "uid": "8FPN-nmik",
-  "version": 3,
-  "description": "Monitor Ballerina HTTP/HTTPs Services, HTTP Clients and SQL Clients"
+  "version": 2
 }
```
Comment (nadundesilva) [inline grafana-dashboard.json:2875]: New line at the end

File: grafana-dashboard.json (hunk 1 above)
Comment (tharinduwijewardane) [reply grafana-dashboard.json]: Turned off thresholds

File: grafana-dashboard.json (hunk 2 above)
Comment (tharinduwijewardane) [reply grafana-dashboard.json]: Fixed

**Review by nadundesilva — APPROVED** [review]
